test/e2e-single-node/sno_mcd_test.go

```diff
@@ -405,9 +406,11 @@ func waitForSingleNodePoolComplete(t *testing.T, cs *framework.ClientSet, pool,
 		return true, nil
 	}); err != nil {
 		if err == wait.ErrWaitTimeout {
-			fmt.Errorf("pool %s is still not updated, waited %v: %v: %w", pool, time.Since(startTime), lastErr, err)
+			errs := kubeErrs.NewAggregate([]error{fmt.Errorf("pool %s is still not updated, waited %v: %w", pool, time.Since(startTime), lastErr), err})
+			fmt.Errorf("%w", errs)
```

Comment:
@cheesesashimi the nesting of errors here as well

Reply:
Same thing here, do something like:

```golang
errs := kubeErrs.NewAggregate([]error{lastErr, err})
if err == wait.ErrWaitTimeout {
  return fmt.Errorf("pool %s is still not updated, waited %v: %w", pool, time.Since(startTime), errs)
}
return fmt.Errorf("unknown error occurred: %w", errs)
```